docs/guidance/evaluating.md

```diff
@@ -30,11 +30,11 @@ Check whether the project-level data is published using [OC4IDS](../projects/ind
 
     .. markdown::
 
-      You can use a [JSON schema validator](https://json-schema.org/implementations.html#validators) to check that whether your data is in the correct format.
+      You can use the [OC4IDS Data Review Tool](http://standard.open-contracting.org/infrastructure/review) to check that whether your data is in the correct format.
```

Comment:
Add trailing slash to URL. Use HTTPS. Don't merge until URL works.

Reply:
Resolved in https://github.com/open-contracting/infrastructure/pull/144/commits/f15dd288f44ed6410de5637d4103b06d3f436fc3